src/app/settings/organization-plans.component.html

```diff
@@ -209,8 +210,16 @@ <h2 class="spaced-header mb-4">{{'paymentInformation' | i18n}}</h2>
         </ng-container>
         <small class="text-muted font-italic mt-2 d-block" *ngIf="!createOrganization">
             {{'paymentCharged' | i18n : (interval | i18n) }}</small>
-    </ng-container>
-    <div [ngClass]="{'mt-4': !createOrganization || plans[plan].noPayment}">
+    </div>
+    <div class="mt-4">
+        <div id="price" class="my-4" *ngIf="total > 0">
+            <div class="text-muted text-sm">Plan price: {{ subtotal | currency: 'USD $' }}
+                <br />Estimated tax: {{ salesTax | currency: 'USD $' }}</div>
```

Comment:
I am wondering if we should toggle this off until an API is chosen and implemented? Right now it just shows on screen that sales tax is 0$.

Reply:
This needs to be visible only if the selected billing country is the US. Otherwise we would leave it, even if it's always zero.

Also, we should be using our own API for doing this, so the UI would never know when the server is going to start returning a value other than zero.

Reply:
Removed